### DISCUSSION TITLE
4th order pde

### DISCUSSION THREAD

**Discussion by divyanshjnv1** [lululxvi/deepxde#1712] [Q&A]
Dear All,
I want to solve the **4th order PDE**  ( Cahn Hilliard Phase field equation ) using DEEPXDE. Solution doesn't seem to converge correctly.
What strategies can be used for improving convergence, what should I keep in mind while trying for 4th order PDE. 
Please provide your general thoughts too.  I am all ears.
thanks!

**Comment by xuliang5115** [comment]
1. Try more iterations;
2. Normalized input (because the activation function performs well between [-1, 1]);
3. Add a feature transform (`net.apply_feature_transform(func)`);
4. Try loss weights...

**Reply by divyanshjnv1** [reply]
Thanks a lot. I'll try that.
can you explain a bit how feature transform help in improving convergence.
